app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str) -> None:
+        super().__init__(nickname)

```

Comment:
The `super().__init__(nickname)` call is incorrect because the `Player` class does not have a constructor that accepts a `nickname` parameter. You need to ensure that the `Player` class has an appropriate constructor or modify this call to match the `Player` class's constructor.

Reply:
thats because he found a player at __init__ file...